### DISCUSSION TITLE
[Feature] Hide irelevant parent folder (${External_Path} variable) for external library on Folders page

### DISCUSSION THREAD

**Discussion by laurfb** [immich-app/immich#13344] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi.
Perhaps it would be useful/simpler if the display of the folders for the external library did not contain the entire path but only the final folders, grouped under the generic name "External library". Like uploads folders.
Without the full mount points path specified in docker_compose.yaml file.
In my case "usr/src/app/".
regards,
Laur
![immich_folders](https://github.com/user-attachments/assets/17669131-9343-4807-97c8-7394bb6b1ac4)
![compose_ext_path](https://github.com/user-attachments/assets/affb1f93-dd88-4e5c-980a-564edadcd7f4)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by petersuba** [comment]
I think this is the same as the comments made on the release note when the folder view was implemented:
https://github.com/immich-app/immich/discussions/12151#discussioncomment-10506072
